about.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>About</title>
+</head>
+
+<body>
+  <header id="about-header">
+    <nav>
+      <div class="wrap top-menu-wrap display-none">
+        <nav class="top-menu">
+          <ul>
+
+            <li>
+              <a href="#">
+                <img src="Images/top-menu-logos/facebook-logo.png" alt="Facebook logo">
+              </a>
+            </li>
+            <li>
+
+              <a href="#">
+                <img src="Images/top-menu-logos/twitter-logo.png" alt="Twitter logo">
+              </a>
+            </li>
+            <li>
+
+              <a href="#">
+                Logout
+              </a>
+            </li>
+            <li>
+
+              <a href="#">
+                My Page
+              </a>
+            </li>
+            <li>
+
+              <a href="#">
+                English
+              </a>
+            </li>
+
+          </ul>
+        </nav>
+      </div>
+
+      <div class="mobile-logo-wrap">
+        <div>
+          <a href="index.html">
+            <img src="Images/main-logo.png" alt="developer week global">
+          </a>
+        </div>
+      </div>
+      <div class="mobile-menu-wrap">
+        <div class="mobile-menu-button">
+          <a href="#" id="mobile-menu-button">
+            <img src="Images/mobile_menu.png" alt="Mobile menu button">
+          </a>
+        </div>
+
+        <div class="mobile-menu-display display-none" id="mobile-menu-display">
+          <div class="mobile-menu-item align-self-fx-end">
+            <button id="close-mobile-menu">&#10799;</button>
+          </div>
+
+          <div class="mobile-menu-item">
+            <a href="#">
+              <p>News</p>
+            </a>
+          </div>
+
+          <div class="mobile-menu-item">
+            <a href="#">
+              <p>Sponsor</p>
+            </a>
+          </div>
+
+          <div class="mobile-menu-item">
+            <a href="#">
+              <p>Join</p>
+            </a>
+          </div>
+
+          <div class="mobile-menu-item">
+            <a href="#">
+              <p>Program</p>
+            </a>
+          </div>
+
+          <div class="mobile-menu-item">
+            <a href="about.html">
+              <p>About</p>
+            </a>
+          </div>
+        </div>
+      </div>
+
+      <div class="wrap header-wrap">
+        <div class="main-menu">
+
+          <div class="header-content">
+            <div class="main-logo">
+              <a href="index.html">
+                <img src="Images/main-logo.png" alt="developer week global">
+              </a>
+            </div>
+            <div class="menus">
+              <ul>
+                <li class="menu-item"><a href="about.html">About</a></li>
```

Comment:
It's almost done, but the `CC Campaign` button is missing on the right. And BTW don't you think the menu is aligned to the right? 🤔

![image](https://user-images.githubusercontent.com/34779928/197263895-8f4f6a7d-a10d-4ecd-8f9f-cfc2349f76ed.png)